components/collective-page/hero/HeroAvatar.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import dynamic from 'next/dynamic';
 import styled, { css } from 'styled-components';
 import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
 import { Mutation } from 'react-apollo';
+import { useDropzone } from 'react-dropzone';
```

Comment:
`react-dropzone` should stays imported dynamically here (as it was before the update). We don't want to make the bundle heavier for everyone because only admin users will need this.